app/controllers/news_items_controller.rb

```diff
@@ -190,20 +194,32 @@ def create_params
   end
 
   def convert_params_for_graphql
-      # Convert has_many content_blocks(which has_many media_contents)
-      content_block_params = @news_item_params["content_blocks"]
-      return unless content_block_params.present?
-
-      content_blocks = []
-      content_block_params.each do |key, content_block|
-        next if content_block.blank?
-        media_contents = []
-        content_block[:media_contents].each do |key, media_content|
-          media_contents << media_content
-        end
-        content_block[:media_contents] = media_contents
-        content_blocks << content_block
+    # Convert has_many dates
+    if @news_item_params["categories"].present?
+      categories = []
+      @news_item_params["categories"].each do |key, category|
+        next if category.blank?
+
+        categories << category
+      end
+      @news_item_params["categories"] = categories
```

Comment:
is es nich am ende das gleiche wie vorher?
alle categories aus params durchgehen und dann wieder als params setzen?

Reply:
die leeren Elemente fliegen raus